### DISCUSSION TITLE
v.1.3.x is Under Celebrated

### DISCUSSION THREAD

**Discussion by poirazis** [Budibase/budibase#7666] [General]
The v.1.3.x release is seriously under celebrated. A smooth rollout, packed with goodies and fixes, and the so awaited clickable Container by @melohagan.

Way to go guys! 
Big big thank you!

**Comment by melohagan** [comment]
Well big shout out to @aptkingston for getting the clickable containers over the line 🚀 

Thanks for the kind words 😍

**Comment by JayP718** [comment]
Agreed, the show alerts actions was an awesome much needed feature.

**Comment by pandel** [comment]
May I ask, where you got the detailed information about the content of the releases? Clickable containers and the show alerts function are totally new to me. The releases page is not very informative, and the tag comparison does not reveal anything in this regard, too, at least for me: https://github.com/Budibase/budibase/compare/v1.3.0...v1.3.11

So, how did you find these new features?

**Reply by poirazis** [reply]
Compare the tags 1.2.52 and 1.3.0

**Reply by pandel** [reply]
Ok, did that and I roughly understand what we can see here. This not a simple 1,2,3 list of new features, but that's what I thought of. I can't celebrate what I don't know about ;-)

**Comment by MentalGear** [comment]
Seems to me like the whole Budibase is under-celebrated ! I for example had to search quite a bit for retool alternative until finding this open-source all-in-one (hidden) gem - Huge APPLAUSE to the whole FOS development team! 👏👏🎉🎉
